.gitignore

```diff
@@ -22,4 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
-.env
\ No newline at end of file
+.env
+server/env
+client/env
```

Comment:
like I mentioned in the previous review, you shouldn't have any files called `env`, only `.env`, this doesn't need to be in the .gitignore

Reply:
Deleted the .env files